### PR TITLE
Remove hardcoded version + MIT

### PR DESCRIPTION
**Pull request by thomas-kl1**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
     "description": "Improve the performance of your Magento 2 store using Barbajs",
+    "keywords": [
+        "barbajs",
+        "magento",
+        "magento2",
+        "module"
+    ],
     "type": "magento2-module",
-    "license": "GPL-3.0",
-    "version": "1.0.0",
```
Comment (mehdichaouch) [inline composer.json:6]: @thomas-kl1 version this is necessary for composer !

Comment (thomas-kl1) [reply composer.json:6]: No ideally we should use git tag to handle versions

Comment (mehdichaouch) [reply composer.json:6]: ok, I always use both, because when using a zip version of repository you don't know version of module.

Comment (thomas-kl1) [reply composer.json:6]: from experience, if it's not automated by a script/hook, this lead to many human errors

**Pull request merged**